impbk2002/Tetra
# refactor MapTask family and other extensions

1. included basic unitest for MapTask and TryMapTask
2. try to move extension method to own namespace to prevent possible conflict
3. use Swift 5.9 as minimum target
4. add tests to verify MultiMapTask
5. refactor MapTask Family for more readability
